src/main/kotlin/model/HMPPSDomainEvents.kt

```diff
@@ -0,0 +1,57 @@
+package uk.gov.justice.hmpps.architecture
+
+import com.structurizr.model.*
+import com.structurizr.view.AutomaticLayout
+import com.structurizr.view.ViewSet
+import uk.gov.justice.hmpps.architecture.annotations.Tags
+
+class DomainEvents private constructor() {
+  companion object : HMPPSSoftwareSystem {
+    lateinit var system : SoftwareSystem
+
+    override fun defineModelEntities(model: Model) {
+
+      system = model.addSoftwareSystem(
+        "hmpps-domain-events",
```

Comment:
Based on its name, could we rename the variable, please, so that its usage in other models is easier to read?

Currently, it'd be `DomainEvents.system`. What do you think about `DomainEvents.topic`?

Reply:
Haha - I've lost track of how many times I've renamed it back and forth - I left it as HMPPSDomainEvents because thats the name we all know it as.